src/components/ItemModal.ts

```diff
@@ -27,6 +27,38 @@ export class ItemModal extends Modal {
     return this.isValidName ? "" : "This is an invalid name. Name's can't include /, \\, :, or .";
   }
 
+  checkIfDatesEmpty(date_setting: string ){
+    if(date_setting == 'Do Date')
+        return this.successPlanItem.do_date != "" ? this.successPlanItem.do_date._i : "";
+    else if(date_setting == 'Due Date')
+        return this.successPlanItem.due_date != "" ? this.successPlanItem.due_date._i : "";
+    else 
+        return this.successPlanItem.closing_date != "" ? this.successPlanItem.closing_date._i : "";
+  }
```

Comment:
For this method, you could have used a switch statement. Additionally, you could have created a method that took in the date property and then called it for each situation.